xaitk_saliency/impls/saliency/fsal.py

```diff
@@ -0,0 +1,65 @@
+from xaitk_saliency import ImageClassifierSaliencyMapGenerator
+from xaitk_saliency.utils.masking import weight_regions_by_scalar
+
+import numpy as np
+
+
+class Fsal (ImageClassifierSaliencyMapGenerator):
```

Comment:
I'm not sure I would call this implementation Fsal, since this is pretty Kitware-specific. It essentially is an occlusion-based saliency algorithm, so maybe something like Occlusion instead?

Reply:
Agreed, I changed it to OcclusionScoring.